__tests__/app.test.js

```diff
@@ -37,4 +37,22 @@ describe("GET /api/topics", ()=>{
             })
         })
     })
+describe("GET /api",()=>{
+    test('200: responds with an object describing all the availbale endpoints on the API', ()=>{
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({body})=>{
+            const {endpoints} = body
+            console.log(endpoints)
+            Object.keys(endpointsValue).forEach((endpointKey)=>{expect(endpoints[endpointKey]).toEqual(endpointsValue[endpointKey])
+                
```

Comment:
its not the greatest test as you're just comparing it to itself - but i'd leave it for now.